main/timeseries.py

```diff
@@ -84,6 +84,44 @@ def get_effort_timeseries(start_date: datetime, end_date: datetime) -> pd.Series
     return timeseries
 
 
+def get_charged_effort_timeseries(
```

Comment:
I may be getting confused but I think as this timeseries is using the effort_per_day property this isn't the _charged_ project effort (effort per day is just total project effort based on the total budget divided by number of working days)